src/luaFunctionList.json

```diff
@@ -0,0 +1 @@
)","lowerWindow":"lowerWindow(labelName)","addCustomLine":"addCustomLine(roomID, id_to, direction, style, color, arrow)","setBorderColor":"setBorderColor(r,g,b)","setBorderBottom":"setBorderBottom(size)","moveWindow":"moveWindow( name, x, y )","resetRoomArea":"resetRoomArea (areaID)","setBgColor":"setBgColor([windowName], r,g,b )","getCurrentLine":"getCurrentLine()","setBackgroundColor":"setBackgroundColor(labelName, red, green, blue, transparency)","moveGauge":"moveGauge(gaugeName, newX, newY)","setAreaName":"setAreaName(areaID or areaName, newName)","getButtonState":"getButtonState()","setLabelOnLeave":"setLabelClickCallback(labelName, luaFunctionName, optional any amount of arguments)","sendSocket":"sendSocket(data)","echoPopup":"echoPopup([window], text, {commands}, {hints}, [current or default format])","deselect":"deselect([optional window name])","permKey":"permKey(name, parent, <i>modifier</i>, key code, lua code)","sendGMCP":"sendGMCP(command)","appendCmdLine":"appendCmdLine()","send":"send( command, show on screen )","selectString":"selectString( [windowName], text, number_of_match )","createMapImageLabel":"labelID = createMapImageLabel(areaID, filePath, posx, posy, posz, width, height, zoom, showOnTop)","getProfileName":"name = getProfileName()","setRoomArea":"setRoomArea(roomID, newAreaID or newAreaName)","saveMap":"saveMap(location)","roomExists":"roomExists(roomID)","getIrcNick":"getIrcNick()","isPrompt":"isPrompt()","highlightRoom":"highlightRoom( roomID, color1Red, color1Green, color1Blue, color2Red, color2Green, color2Blue, highlightRadius, color1Alpha, color2Alpha)","getMainConsoleWidth":"getMainConsoleWidth()","echoUserWindow":"echoUserWindow(windowName)","setUnderline":"setUnderline(windowName, bool)","getRoomIDbyHash":"roomID = getRoomIDbyHash(hash)","setMergeTables":"setMergeTables(module)","getLastLineNumber":"getLastLineNumber(window)","getAreaExits":"getAreaExits(areaID, showExits)","getAreaRooms":"getAreaRooms(area id)","moveCursorEnd":"moveCursorEnd( windowName )","addMapEvent":"addMapEvent(uniquename, event name, parent, display name, arguments)","getColorWildcard":"getColorWildcard(ansi color number)","createLabel":"createLabel(name, Xpos, Ypos, width, height, fillBackground)","disableScrollBar":"disableScrollBar(windowName)","restartIrc":"restartIrc()","hideWindow":"hideWindow(name)","lockRoom":"lockRoom (roomID, lock = true/false)","sendATCP":"sendATCP(message, what)","sendMSDP":"sendMSDP(variable[, value][, value...])","getLineCount":"getLineCount()","wrapLine":"wrapLine(windowName, lineNumber)","getRoomName":"roomName = getRoomName(roomID)","getColumnCount":"getColumnCount([window])","disableKey":"disableKey(name)","getMudletLuaDefaultPaths":"getMudletLuaDefaultPaths()","hecho":"hecho(window, text)","getModulePriority":"priority = getModulePriority(module name)","alert":"alert(<i>seconds</i>)","setLabelDoubleClickCallback":"setLabelDoubleClickCallback(labelName, luaFunctionName, optional any amount of arguments)"}
```

Comment:
Could you pretty-print it and name `lua-function-list.json` to be in line with the rest of resources?